Firestore/Example/Tests/SpecTests/FSTMockDatastore.m

```diff
@@ -76,10 +72,15 @@ - (instancetype)initWithDatabase:(FSTDatabaseInfo *)database
 
 #pragma mark - Overridden FSTWatchStream methods.
 
-- (void)start {
+- (void)start:(id<FSTWatchStreamDelegate>)delegate {
```

Comment:
Labels should always reflect the arguments passed, so this should be `startWithDelegate:`.

Reply:
Done